build.gradle

```diff
@@ -156,14 +156,9 @@ project(':api') {
         }
     }
 
-    task sourcesJar(type: Jar) {
-        classifier = 'sources'
-        from sourceSets.main.allJava
-    }
-
-    task javadocJar(type: Jar, dependsOn: javadoc) {
-        classifier = 'javadoc'
-        from javadoc.destinationDir
+    java {
+        withJavadocJar()
+        withSourcesJar()
```

Comment:
Does the same as before bu uses Gradle's built-in tasks.

Reply:
Actually this is needed @marcphilipp